data/human/outfits.txt

```diff
@@ -394,6 +394,16 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Old Man's Sword"
+	category "Unique"
+	cost 120000
+	thumbnail "outfit/old sword"
+	"capture attack" 3
+	"capture defense" 3
```

Comment:
This sword would be one of the best weapons in the game, only Heliarch weapons and the Void Rifle are better. I'd make it go under the Pug Peacekeeping Staff, after all I don't expect a textile worker from New Boston learnt fencing when they were younger. I'm still thinking about that energy bolt thing.